LQResultProducer.py

```diff
@@ -1,4 +1,4 @@
-import os, sys, math, random, platform, time
+import os, sys, math, random, platform, time, re
```

Comment:
is the 're' real? or added by mistake? what does it do?

Reply:
This is used in line 8112, to call the re.search method. I have the makeOptPlotForPAS function take the mass points from the cutfile instead of hardcoding the masses (for "cutoff" plots the mass points change, so it is more flexible this way).